assets/styles.css.liquid

```diff
@@ -260,7 +260,7 @@ hr.divider { margin: 15px 0; background: {{ settings.border_color }}; height: 1p
     text-transform: uppercase;
 }
 /* 2.2 - Grid System */
-.wrapper { width: 996px; margin: 0 auto; padding: 0 15px; background: {{ settings.shop_bg_color_content_area }}; }
+.wrapper { width: 1026px; margin: 0 auto; padding: 0 15px; background: {{ settings.shop_bg_color_content_area }}; }
```

Comment:
You shouldn't be editing their liquid css file. Best to use fugly selectors or remove all together.